cns/middlewares/k8sSwiftV2.go

```diff
@@ -128,6 +128,12 @@ func (m *K8sSWIFTv2Middleware) validateIPConfigsRequest(ctx context.Context, req
 		if mtpnc.Status.PrimaryIP == "" || mtpnc.Status.MacAddress == "" || mtpnc.Status.NCID == "" || mtpnc.Status.GatewayIP == "" {
 			return nil, types.UnexpectedError, errMTPNCNotReady.Error()
 		}
+		interfaceInfos := mtpnc.Status.InterfaceInfos
+		for _, interfaceInfo := range interfaceInfos {
+			if interfaceInfo.NICType == string(cns.BackendNIC) {
+				req.BackendInterfaceMacAddress = interfaceInfo.MacAddress
```

Comment:
Since at the moment we are only having one IB at most, this would work, but I would assert that we just find at most one of its kind or fail since later this could be and issue hard to itentify

Reply:
Addressed